src/Symfony/Component/Console/Helper/QuestionHelper.php

```diff
@@ -402,7 +402,7 @@ private function mostRecentlyEnteredValue(string $entered): string
     private function getHiddenResponse(OutputInterface $output, $inputStream, bool $trimmable = true): string
     {
         if ('\\' === \DIRECTORY_SEPARATOR) {
-            $exe = __DIR__.'/../Resources/bin/hiddeninput.exe';
+            $exe = '"'.__DIR__.'/../Resources/bin/hiddeninput.exe'.'"';
```

Comment:
Maybe better to add the doubles quotes inside the `shell_exec('"'.$exe.'"')`?

Because you also have the `sys_get_temp_dir().'/hiddeninput.exe'` just after.

Reply:
Good remark! I changed it and tested it locally.